src/BugsnagUnity/Configuration.cs

```diff
@@ -138,6 +138,8 @@ public virtual bool AutoTrackSessions
 
         public ErrorTypes[] EnabledErrorTypes { get; set; }
 
+        public int AppHangThresholdMillis { get; set; } = -1;
```

Comment:
This should probably be `ulong`, to match the precision of [`NSUInteger`](https://developer.apple.com/documentation/objectivec/nsuinteger).

Reply:
ulong can't be -1, how about we split the difference and go for a long?

Reply:
perhaps i sould also add a limit like MaximumBreadcrumbs

Reply:
The [threshold on iOS must be > 250](https://github.com/bugsnag/bugsnag-cocoa/blob/b83847c74669f64a9afc0e7e7ade595b620c09e8/Bugsnag/include/Bugsnag/BugsnagConfiguration.h#L210-L220), so maybe zero could be the sentinel value?

Reply:
Done 👍